### PR TITLE
Call Bootstrap make_col with column number for accurate percentages

### LINKED ISSUE #9032: _widgets.scss math error
For the 33% and 66% columns, the current code calls `make-col` with a decimal value, but it should pass the "bootstrap" col number.

The result is that 33% and 66% end up as 33% and 66%.

If you try to kill the gutters, and make a "full-width" column, you'll see the issue. With gutters, this issue is negligible.

The fix is here:

```
@each $wsize in (25, 33, 50, 66, 75) {
    .widget-size-#{$wsize} {
        @extend .widget-size-100;

        @include media-breakpoint-up(sm) {
            @include make-col(round($wsize*12/100));
        }
    }

    .widget .widget-size-#{$wsize} {
        @extend .widget-size-#{$wsize};

        @include media-breakpoint-up(md) {
            @include make-col(round($wsize*12/100));
        }
    }
}
```

### PR DESCRIPTION
**Pull request by kdubious**
Fixes #9032

### CODE REVIEW

**Comment by Skrypt** [comment]
Needs to be also changed in other themes where we have this _widgets.scss file. And you also need to do a `gulp build` to build the resulting .min.css and .css files.

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=9048) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=9048) before we can accept your contribution.<br/><br/>:x: kdubious [sign now](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=9048)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/OrchardCMS/OrchardCore?pullRequest=9048) it.</sub>

**Comment by kdubious** [comment]
@Skrypt 45 minutes and some cleanup later.

**Comment by agriffard** [comment]
When you want to link to an issue and specify that it will be automatically closed by your PR, use `Fixes #ISSUE_NUMBER`.

You can re-edit your description in the first comment.

**Comment by Skrypt** [comment]
Too much committed files. Like I said, you need to filter files that are necessary to be changed in the repository.
Don't merge this yet.

**Comment by kdubious** [comment]
@Skrypt why is the gitignore not filtering the things you don’t want me to commit?

File: src/OrchardCore.Themes/TheComingSoonTheme/wwwroot/package-lock.json [hunk 1]
```diff
@@ -1,7203 +1,8 @@
 {
     "name": "startbootstrap-coming-soon",
     "version": "5.0.9",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheComingSoonTheme/wwwroot/package-lock.json:4]: This should stay to version 2. You need to upgrade your npm version to latest or node.js.

File: src/OrchardCore.Themes/TheBlogTheme/wwwroot/package-lock.json [hunk 2]
```diff
@@ -1,7273 +1,8 @@
 {
     "name": "startbootstrap-clean-blog",
     "version": "5.0.10",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheBlogTheme/wwwroot/package-lock.json:4]: Same thing

File: src/OrchardCore.Themes/TheComingSoonTheme/Assets.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "generateSourceMaps": false,
+    "inputs": [ "wwwroot/scss/bootstrap-oc.scss" ],
+    "output": "wwwroot/css/bootstrap-oc.css"
+  }
+]
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheComingSoonTheme/Assets.json:7]: We don't want to build themes on every `gulp build` this is why you need to run it directly on each wwwroot folder of each of these themes. At least for now keep it but add an extra line at the end of file.

File: src/OrchardCore.Themes/TheBlogTheme/Assets.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "generateSourceMaps": false,
+    "inputs": [ "wwwroot/scss/bootstrap-oc.scss" ],
+    "output": "wwwroot/css/bootstrap-oc.css"
+  }
+]
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheBlogTheme/Assets.json:7]: new line at eof

**Comment by kdubious** [comment]
Rolled back the package-lock changes and added the new lines to Assets.json

**Comment by kdubious** [comment]
@Skrypt Better?

File: src/OrchardCore.Themes/TheBlogTheme/wwwroot/scss/bootstrap-oc.scss [hunk 5]
```diff
@@ -13,6 +13,7 @@ By Orchard Core Team
 @import "modules/_pager.scss";
 @import "modules/_taxonomy.scss";
 
+
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheBlogTheme/wwwroot/scss/bootstrap-oc.scss]: revert

File: src/OrchardCore.Themes/TheTheme/wwwroot/scss/bootstrap-oc.scss [hunk 6]
```diff
@@ -1,4 +1,5 @@
 @import "../../node_modules/bootstrap/scss/bootstrap";
+
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheTheme/wwwroot/scss/bootstrap-oc.scss]: revert

File: gulpfile.js [hunk 7]
```diff
@@ -22,9 +22,9 @@ var fs = require("graceful-fs"),
     postcss = require('gulp-postcss'),
     rtl = require('postcss-rtl'),
     babel = require('gulp-babel');
-    // The startbootstrap theme gulpfiles do not yet support node 15 so need to be installed and built seperately.
-    // blogtheme = require('./src/OrchardCore.Themes/TheBlogTheme/wwwroot/gulpfile');
-    // comingsoontheme = require('./src/OrchardCore.Themes/TheComingSoonTheme/wwwroot/gulpfile');
+// The startbootstrap theme gulpfiles do not yet support node 15 so need to be installed and built seperately.
```
Comment (Skrypt) [inline gulpfile.js:25]: Revert the changes on this file. We will asses the gulpfile issues in a different PR.

File: gulpfile.js [hunk 8]
```diff
@@ -72,64 +72,88 @@ gulp.task("watch", function () {
...
+
+//gulp.task("build-comingsoontheme", function (done) {
+//    return buildComingSoonTheme(done);
+//});
+
+//gulp.task("build-theme", function (done) {
+//    return buildTheme(done);
+//});
+
+//gulp.task("build-themes", gulp.parallel('build-agencytheme', 'build-blogtheme', 'build-comingsoontheme', 'build-theme'));
```
Comment (Skrypt) [inline gulpfile.js:112]: This unnecessary to comment these out if they are not used anywhere.

File: gulpfile.js [hunk 9]
```diff
@@ -72,64 +72,88 @@ gulp.task("watch", function () {
...
 
 
 // gulp.task( 'build',  gulp.series([ 'build-assets', 'build-themes' ]) );
 // gulp.task( 'rebuild',  gulp.series([ 'rebuild-assets', 'build-themes' ]) );
```
Comment (Skrypt) [inline gulpfile.js:116]: Here is where it was used

File: src/OrchardCore.Themes/TheComingSoonTheme/Assets.json [hunk 10]
```diff
@@ -0,0 +1,7 @@
+[
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheComingSoonTheme/Assets.json]: If we don't build these themes from the main gulp pipeline what are these for?

File: gulpfile.js (hunk 7 above)
Comment (Skrypt) [reply gulpfile.js:25]: Remove these commented lines and the comment line.

File: gulpfile.js (hunk 9 above)
Comment (Skrypt) [reply gulpfile.js:110]: Removee these commented lines

File: gulpfile.js [hunk 11]
```diff
@@ -72,64 +72,64 @@ gulp.task("watch", function () {
...
         process.chdir(cwd);
         done();
     });
 }
 
-function buildComingSoonTheme(done){
-    var cwd = process.cwd();      
-    process.chdir('./src/OrchardCore.Themes/TheComingSoonTheme/wwwroot');    
-	comingsoontheme.build( ()=> {
+function buildComingSoonTheme(done) {
```
Comment (Skrypt) [inline gulpfile.js:129]: These functions can be removed too

**Comment by kdubious** [comment]
@Skrypt I think the gulpfile is back to how I found it, no?

**Comment by Skrypt** [comment]
If it was like it was before there would be no diff on the file.

**Comment by kdubious** [comment]
@Skrypt Agreed, and the differences are:
1. White space
2. removing 3 commented lines from the original (that I think you asked me to remove)

**Comment by Skrypt** [comment]
I'm trying this locally and I get errors while trying to do `gulp build`

image

**Review by Skrypt — APPROVED** [review]
